src/components/device-screen.tsx

```diff
@@ -11,84 +11,242 @@ import { Devices } from '../lib/xively/models/index';
 
 import { View, Text, Button, Image, ScrollView } from "react-native";
 import Styles from '../styles/main';
+import { VictoryChart, VictoryLine, VictoryVoronoiContainer, VictoryTooltip, VictoryAxis } from "victory-native";
```

Comment:
similar to how it is in RIoT I would like to have the timeseries chart abstracted into its own control.  It would be ideal to have exactly the same params on the control as is in the web RIoT